src/Task/Development/loadTasks.php

```diff
@@ -9,7 +9,7 @@
      */
     protected function taskChangelog($filename = 'CHANGELOG.md')
     {
-        return new Changelog($filename);
+        return new Changelog($filename, $this);
```

Comment:
Is that really needed to bind to class? 
In `GenMarkdown` we also use closures everywhere but they are not bound to anything. I don't see any real necessity in this but it adds additional dependency to a task, so I don't see as the most clear solution